### DISCUSSION TITLE
Sending json data to js clients

### DISCUSSION THREAD

**Discussion by etempm** [jchristn/WatsonWebsocket#131] [Q&A]
Hi

Is it possible to use websocket server as a json api for javascript web client as datasource ? Maybe 2 - 3 kb json data. What is your suggestion?

Thanks for this library, it is realy easy and helpfull.

**Comment by jchristn** [comment]
Hi @etempm thanks for your note.  I'm not sure I understand 100%.  You want to use JS in a web client (browser) consuming data from a websocket server (based on WatsonWebsocket) - is this correct?

If so, yes, absolutely, it's quite easy to do so.  The README gives a simple implementation and also a reference on how to connect to the websocket server using a Javascript web client.  Also you can check out the ```Test.Server``` package and modify that to put something together quickly to suit your needs.

**Reply by etempm** [reply]
Thanks for quick response. I will test and share the results.

**Comment by etempm** [comment]
It works like a charm. Thanks. Bu I have another problem, maybe bug. I am posting in Issues.
